### PR TITLE
[Win] removed some warnings

### PR DESCRIPTION
**Pull request by wargio**
fixed dll redeclaration
fixed warnings regarding ULLFMT for json.c & query.c

### CODE REVIEW

File: src/json.c [hunk 1]
```diff
@@ -280,7 +280,11 @@ SDB_API const char *sdb_json_format(SdbJsonString* s, const char *fmt, ...) {
 			case 'l':
 				JSONSTR_ALLOCATE (32);
 				arg_l = va_arg (ap, ut64);
+#if __SDB_WINDOWS__ && !__CYGWIN__
+				snprintf (tmp, sizeof (tmp), "0x%llx", arg_l);
```
Comment (radare) [inline src/json.c]: this is not correct, ULLFMT should be fixed in src/types.h instead

Comment (wargio) [inline src/json.c]: Ok!

**Comment by radare** [comment]
waiting for the updated pr

**Comment by radare** [comment]
please rebase and update your pr fixing my comments

**Comment by wargio** [comment]
i'll create a new one.

**Pull request closed without merging**